sound/soc/intel/boards/sof_sdw.c

```diff
@@ -555,6 +555,7 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.direction = {true, true},
 				.dai_name = "rt700-aif1",
 				.dai_type = SOF_SDW_DAI_TYPE_JACK,
+				.dailink = {0, 1},
```

Comment:
commit message: you should clarify if there's any change in behavior or just the same results implemented differently.